Assets/Mirror/Runtime/Transport/MultiplexTransport.cs

```diff
@@ -24,6 +24,12 @@ public void Awake()
             InitServer();
         }
 
+        public override bool Available()
+        {
+            // the whole point of Multiplexing is to be available everywhere
+            return true;
```

Comment:
Would be better to loop through the transports and return true if one of them is available

Reply:
true, fixed